data/favicon.json

```diff
@@ -49,6 +49,7 @@
 	"buhl-newsletter.de": "buhl.de.ico",
 	"bund.net": "bund.net.ico",
 	"buyon.com": "buyon.it.ico",
+	"service.cacf-festgeld.de": "service.cacf-festgeld.de.png",
```

Comment:
At the moment, they're not signing mails from the base domain. I can also change this to `cacf-festgeld.de` in case you prefer it. I thought I'd keep the sub domain here and in the according rule.

Reply:
Even if they are currently only signing this sub-domain, specifying the icon for the complete domain would mean if they ever start signing a different sub-domain or the top-domain, the icon would be automatically added there too.